cli/command/system/prune.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/cli"
+	"github.com/docker/docker/cli/command"
+	"github.com/docker/docker/cli/command/prune"
+	units "github.com/docker/go-units"
+	"github.com/spf13/cobra"
+)
+
+type pruneOptions struct {
+	force bool
+	all   bool
+}
+
+// NewPruneCommand creates a new cobra.Command for `docker du`
+func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
+	var opts pruneOptions
+
+	cmd := &cobra.Command{
+		Use:   "prune [COMMAND]",
+		Short: "Remove unused data",
+		Args:  cli.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runPrune(dockerCli, opts)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
+	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images not just dangling ones")
+
+	containerPruneCmd := prune.NewContainerPruneCommand(dockerCli)
+	containerPruneCmd.Use = "containers"
+	volumePruneCmd := prune.NewVolumePruneCommand(dockerCli)
+	volumePruneCmd.Use = "volumes"
+	imagePruneCmd := prune.NewImagePruneCommand(dockerCli)
+	imagePruneCmd.Use = "images"
+
+	cmd.AddCommand(
+		containerPruneCmd,
+		volumePruneCmd,
+		imagePruneCmd,
+	)
```

Comment:
Since people were going to use `docker system df` to see usage, i just assume it'd make sense to have them accessible via `docker system` too.

I guess I can update the description of the command to have the user refer to the individual top level command instead and remove those from here (they are the exact same commands in the end).